<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputseDialog</class>
 <widget class="QDialog" name="inputseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>801</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>50</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>row:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rowspinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>50</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>col:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="colspinBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
